docs/api/version-history.md

```diff
@@ -21,6 +21,7 @@ keywords: "API, Docker, rcli, REST, documentation"
   and `OperatingSystem` if the daemon was unable to obtain this information.
 * `GET /info` now returns information about the product license, if a license
   has been applied to the daemon.
+* `POST /swarm/init` now accepts a `default-addr-pool` property to set global scope default address pool
```

Comment:
This is about the API, so there's two new properties; `DefaultAddrPool` and `SubnetSize`


Can you also update the Swagger file with the new fields https://github.com/moby/moby/blob/master/api/swagger.yaml

Reply:
Got it will update both comment

Reply:
Regarding unit test:  main logic and validation happens in CLI and libnetwork.  I added units tests to cover all valid and invalid scenarios there. In Moby , we just pass params in to Swarm.  Hence I dont think we need any special unit test to validate anything here. CLI and Libnetwork  tests will take care of it.